Symfony/CS/Tests/Fixer/Symfony/NoEmptyLinesAfterPhpdocsFixerTest.php

```diff
@@ -212,4 +212,26 @@ public function testFixesWindowsStyle()
 
         $this->makeTest($expected, $input);
     }
+
+    // empty line between typehinting docs and return statement should be preserved
+    public function testInlineTypehintingDocsBeforeReturn()
+    {
+        $expected = <<<'EOF'
+<?php
+
+function parseTag($tag)
+{
+    $tagClass = get_class($tag);
+
+    if ('phpDocumentor\Reflection\DocBlock\Tag\VarTag' === $tagClass) {
+        /** @var DocBlock\Tag\VarTag $tag */
```

Comment:
Fixing this would probably require my docblock classes. A "correct" fix in my opinion would be if the docblock contains a "var" or "type" annotation, then it must document a variable/property/constant. If it doesn't, then we shouldn't modify lines following it because it's not documenting anything.